x-pack/plugins/reporting/public/directives/export_config/export_config.js

```diff
@@ -56,15 +56,16 @@ module.directive('exportConfig', ($rootScope, reportingDocumentControl, reportin
       $scope.updateUrl = (options) => {
         return $scope.getRelativePath(options)
           .then(relativePath => {
-            $scope.exportConfig.absoluteUrl = createAbsoluteUrl(relativePath);
+            $scope.exportConfig.absoluteUrl = createAbsoluteUrl(relativePath)
+              || i18n('xpack.reporting.exportConfig.generationUrl.loadingPlaceholder', { defaultMessage: 'Loading...' });
```

Comment:
This moved code looks like it needs some manual testing. I'm not super familiar with this file. Do you have testing steps for this?